### PR TITLE
doc: fix variable name error

### PR DESCRIPTION
**Pull request by pareyesv**
## Summary/Motivation:
fix variable name error in documentation

## Changes proposed in this PR:
- fix variable name error in documentation

### Legal Acknowledgement

By contributing to this software project, I have read the [contribution guide](https://pyomo.readthedocs.io/en/stable/contribution_guide.html) and agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the BSD license.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by blnicho — APPROVED** [review]

**Comment by blnicho** [comment]
This is a very minor fix to our online documentation so I'm going to go ahead and merge it without waiting for a second review.

**Review by jsiirola — APPROVED** [review]

**Pull request merged**
